packages/pn-personafisica-webapp/src/__mocks__/Contacts.mock.ts

```diff
@@ -50,6 +50,53 @@ export const digitalAddresses: Array<DigitalAddress> = [
   },
 ];
 
+export const orderedDigitalAddresses: Array<DigitalAddress> = [
```

Comment:
I should remove this mock because this file is designed to simulate the be response